### ISSUE TITLE
Normalize

### ISSUE THREAD

**Issue by xxxdongjiu** [hqh0728/CrossGNN#3]
adj = F.softmax(adj.masked_fill(indices_to_remove, filter_value), dim=0)
This code normalizes the rows, but from the treatment of the adjacency matrix below, the row vectors of the matrix are the set of neighboring nodes of a node, so shouldn't they be normalized to the column dimension, or is the adjacency matrix symmetric, and normalizing to either dimension is fine.
